results/forms/forms101.py

```diff
@@ -75,3 +76,14 @@ def form_01(direction, iss, fwb, doc, leftnone):
     fwb.append(tbl)
 
     return fwb
+
+
+def check_len_result(column_result):
+    if len(column_result) >= 3:
+        return column_result
+    else:
+        column_result.append('')
+        check_len_result(column_result)
```

Comment:
Тут похоже планировалась рекурсия, но нет возврата

Reply:
То есть что то вроде
```suggestion
        return check_len_result(column_result)
```

Reply:
Но эта функция не нужна, напишу выше

Reply:
убрал